packages/inter-protocol/test/smartWallet/contexts.js

```diff
@@ -125,3 +127,30 @@ export const makeDefaultTestContext = async (t, makeSpace) => {
     simpleCreatePriceFeed,
   };
 };
+
+/**
+ * @param {Awaited<ReturnType<typeof coalesceUpdates>>} state
+ * @param {Brand<'nat'>} brand
+ */
+export const purseBalance = (state, brand) => {
```

Comment:
Call this `invitationPurseBalance`?  Since #6807 , won't work on any other brand, right?